buildSrc/src/main/groovy/Classpaths.groovy

```diff
@@ -122,6 +122,10 @@ class Classpaths {
     static final String GUAVA_NAME = 'guava'
     static final String GUAVA_VERSION = '33.2.0-jre'
 
+    static final String AVRO_GROUP = 'org.apache.avro'
+    static final String AVRO_NAME = 'avro'
+    static final String AVRO_VERSION = '1.8.1'
```

Comment:
It looks like we are regressing our previous dep version 1.11.3; is 1.8.1 specifically necessary for hadoop-common/iceberg?

Reply:
No, I just made a mistake.  I misread the numbers and thought I was advancing to the latest. Ooops.